exercises/practice/eight-queens/eight_queens_test.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from eight_queens import eight_queens
+
+class EightQueensTester(unittest.TestCase):
+    def test(self):
+        positions = eight_queens()
+        self.assertEqual(len(set(x for (x, y) in positions)), 8)
+        self.assertEqual(len(set(y for (x, y) in positions)), 8)
+        self.assertEqual(len(set(x+y for (x, y) in positions)), 8)
+        self.assertEqual(sum(x == y for (x, y) in positions), 1)
```

Comment:
Is it possible to add a few more test cases? Or is this all of the ones that work? Sorry, I'm not familiar with chess at all

Reply:
The test case validation checks that whatever z3 returns is a valid choice (and it's strikingly similar to the z3 code). Since there's not really any inputs, I can't think of a way to add more test cases, unless we can make z3 "seed" it's choices and give different outputs.